superset-frontend/src/components/ListView/Filters.tsx

```diff
@@ -63,6 +66,7 @@ const filterSelectTheme: PartialThemeConfig = {
   spacing: {
     baseUnit: 2,
     minWidth: '5em',
+    fontSize: FONT_SIZE,
```

Comment:
```suggestion
    fontSize: ${({ theme }) => theme.typography.sizes.s},
```

Reply:
wait I don't think that works, I need to outright import the theme I think

Reply:
Nope! It should be available via the ThemeProvider way up at the top of the app.

Reply:
In fact, it's _better_ to use the theme through the provider than importing it from the `superset-ui` package. Then we can do fun stuff with it, e.g. `<ThemeProvider theme={...supersetTheme, ...darkMode, ...myCompanyBrandColors}>`

Reply:
wait I'm not in a text string here, this is a good old JS object... I think we'd have to convert those objects to emotion first...

Reply:
Just co comment... addressed this in subsequent commits.